### PR TITLE
GitHub actions badges

### PR DESCRIPTION
**Pull request by oefe**
Replace Travis badge with GitHub Actions badges

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,12 +1,32 @@
-|PyPI| |Travis| |Gitter| |DOI| |binder|
+|PyPI| |Test| |Branca| |Selenium| |Notebook| |Notebook-Style| |Publish-PyPI| |Gitter| |DOI| |binder|
```
Comment (ocefpaf) [inline README.rst:1]: We should probably add only the code tests to the README b/c it is what the contributors/users need to know. The others are more of an internal thing and rarely block a PR from being merged.

Comment (oefe) [reply README.rst]: Yeah, the list got a bit too long.

I'm not sure what you mean with "only the code tests". Surely contributor would want to know if their change breaks a  Selenium test or an example notebook? Or if their change to a notebook follows style conventions? So I left these in, and only removed the Branca and Publish-PyPI badges.

That's still two rows of badges, though

Comment (ocefpaf) [reply README.rst]: > Surely contributor would want to know if their change breaks a Selenium test or an example notebook? 

Yeah but that is shown in the PR status. The badges here are not really for devs/contributors. More like a display case for, everything is green, move along :-)

**Comment by ocefpaf** [comment]
Thanks @oefe! Looks really nice!!

**Pull request merged**